### ISSUE TITLE
Notification blocked by Remote content message

### ISSUE THREAD

**Issue by Yaff1e** [itemir/thundersec#29]
If you have _Allow Remote Content In Message_ disabled in Thunderbird a notification is shown at the top of relevant emails (_To protect your privacy, Thunderbird has blocked remote content in this message_) that covers the Thundersec notification until you clear it. Just wondering if the Thundersec notification could be dropped down a line when this happens please?

**Comment by itemir** [comment]
Hi @Yaff1e, can you elaborate a bit? Thundersec message (at least the red ones) should be on top of (displayed first) when "To protect your privacy, Thunderbird has blocked remote content in this message". 

That is because of notification priorities. RedThundersec notifications have higher priority than a notification with a yellow label. Are those the types of labels you see or are you referring to information gray Thundersec notifications, which by definition should be below yellow notifications.

**Comment by Yaff1e** [comment]
Yes, I've just tried again and you're right. The red ones cover the Blocked Content messages. The blocked Content ones cover the grey Thundersec notificiations. I assumed they would be stacked with both visible.

Thanks for helping.

**Comment by itemir** [comment]
Sure, no problem. Gray notifications are informational only (hence lower priority) while red notifications show some kind of a violation. 

I am closing this ticket.
